templates/partials/_carousel-videos-layout.html

```diff
@@ -0,0 +1,16 @@
+<div class="p-carousel p-carousel--videos u-equal-height">
```

Comment:
It's a bit misleading that CSS class is called `p-carousel` when in case of layout with video we don't have any carousel at all.

Reply:
Updated!